lte/gateway/package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Does this file belong to this PR?

Reply:
Thank you for catching this anomaly. This change was intended for a different branch.